tmt/steps/execute/__init__.py

```diff
@@ -597,6 +610,9 @@ def prepare_tests(self, guest: Guest, logger: tmt.log.Logger) -> list[TestInvoca
 
     def prepare_scripts(self, guest: "tmt.steps.provision.Guest") -> None:
         """ Prepare additional scripts for testing """
+        # Create scripts directory
+        guest.execute(Command("mkdir", "-p", f"{SCRIPTS_DEST_DIR}"))
```

Comment:
`str(SCRIPTS_DEST_DIR)` might be more readable.

Reply:
changed.